hoodie-common/src/main/java/com/uber/hoodie/common/util/ParquetUtils.java

```diff
@@ -146,10 +146,10 @@ public static Schema readAvroSchema(Configuration configuration, Path parquetFil
    * Read out the bloom filter from the parquet file meta data.
    */
   public static BloomFilter readBloomFilterFromParquetMetadata(Configuration configuration,
-      Path parquetFilePath) {
+      Path parquetFilePath, boolean enableDynamicBloomFilter) {
     String footerVal = readParquetFooter(configuration, parquetFilePath,
         HoodieAvroWriteSupport.HOODIE_AVRO_BLOOM_FILTER_METADATA_KEY).get(0);
-    return new BloomFilter(footerVal);
+    return new BloomFilter(footerVal, enableDynamicBloomFilter);
```

Comment:
I am assuming this is needed coz only then we 'd know the type of bloom filter and deserialize properly?Can BloomFilter read a DynamicBloomFilter serialized value? what happens when we have a mix of files written using normal and dynamic filterS? should we resolve this using an additional footer, instead of making this a writer side config

Reply:
Have some thoughts around this, will discuss f2f